### PR TITLE
styles/constants: adding constants/libs to use them for styling later:

### PR DESCRIPTION
**Pull request by khamui**
Added FONTS, COLORS and SPACINGS (based on values of adhocracy+)
Initial setup. To be extended modified

### CODE REVIEW

**Comment by khamui** [comment]
not really sure what naming is consistent, descriptive and flexible at the same time. kinda cryptic now 😵‍💫

**Comment by Rineee** [comment]
> not really sure what naming is consistent, descriptive and flexible at the same time. kinda cryptic now 😵‍💫

Haha, we would need @phillimorland for this :) But I think I would prefer the sm-md-lg naming, because it is consistent with our other naming.. what do you think? Or do you think its not flexible enough?

File: theme/colors.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    main: '#1a1a1a',
+    inverted: this.paper.main
+  },
+  grey: {
+    extralight: '#d8d8d8',
+    light: '#4d4d4d',
+    medium: '#666',
+    dark: '#464a4c',
+  },
+};
```
Comment (goapunk) [inline theme/colors.js:28]: not sure how this is handled in the other projects, but maybe add a newline at the end of this line (same for the other two files)?

**Review by fuzzylogic2000 — COMMENTED** [review]
Nice! But do we need this many differnt spacings and font sizes? Can we start with a smaller set and only add more when we absolutely need to?

File: theme/colors.js (hunk 1 above)
Comment (phillimorland) [reply theme/colors.js:28]: our linter automatically saves a blank last line, I can look into our linter configs and see what can be reused here.

Comment (khamui) [reply theme/colors.js:28]: i can add the rule `eol-last` to it, that will make sure any file ends with a newline (think that is what we used in our other linting as well)

**Comment by khamui** [comment]
okay will reduce the list. makes sense to start with as least as possible!

**Review by phillimorland — APPROVED** [review]
Nice! :)

**Pull request merged**